bamboo/Dockerfile

```diff
@@ -1,6 +1,13 @@
 ## Dockerfile to create integration/unit test environment
-FROM node:10.16.3
+FROM node:10.16.3-buster
```

Comment:
Not that it matters, but I'm curious what the `buster` image gives us

Reply:
Python 3.6+ (3.7 specifically).      Swapping via pyenv was a bit too brittle for CI and this handles the problem nicely, given node releases such a wide array of underlying images